client/coral-embed/index.js

```diff
@@ -0,0 +1,163 @@
+/* eslint-disable no-var */
+/**
+ * @file Exposes the main 'window.Coral.Talk' API that developers can use to
```

Comment:
I think if we don't further process this file, the LICENSE of pym.js may require me to copy paste it into this module.
https://github.com/nprapps/pym.js/blob/master/LICENSE